phpBB/includes/acp/acp_board.php

```diff
@@ -530,11 +530,13 @@ function main($id, $mode)
 
 			if ($submit)
 			{
-				if (strpos($data['type'], 'password') === 0 && $config_value === '********')
+				if (strpos(isset($data['type']) ? $data['type'] : '', 'password') === 0 && $config_value === '********')
```

Comment:
if (strpos($data['type'] ?? '', 'password') === 0 && $config_value === '********')

is the same, but maybe more simple

Reply:
Correct, the coalesce is simpler.